frontend/server/src/main/java/com/amazonaws/ml/mms/util/ConfigManager.java

```diff
@@ -167,6 +169,10 @@ public String getHostName() {
         return hostName;
     }
 
+    public void setMmsDecodeInputRequest(String set) {
```

Comment:
We can remove this and use configManager.setProperty() from within tests.

Reply:
Couldn't use setProperty. Just used reflection to achieve the same.